gensim/models/word2vec_inner.pxd

```diff
@@ -1,14 +1,65 @@
+# distutils: language = c++
```

Comment:
need move this functionality to distinct file for avoiding compilation issue on win or old compilers (because almost all *2vec depends on `word2vec_inner`